backend/server3.js

```diff
@@ -0,0 +1,206 @@
+var express = require('express');
+var app = express();
+
+const bodyParser = require('body-parser');
+const { application } = require('express');
+
+var util = require('util');
+var encoder = new util.TextEncoder('utf-8');
+const { MongoClient } = require("mongodb");
+const { response } = require("express");
+const uri = "mongodb://127.0.0.1:27017"
+const client = new MongoClient(uri)
+
+//for accessing other apis
+const request = require('request');
+app.use(express.json());
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const OneSignal = require('@onesignal/node-onesignal') ;
+
+const app_key_provider = {
+    getToken() {
+        return 'MjJiMGQ3ODAtNjIzNC00ZjBkLTgxNGYtMDY0M2YxYWJlNjE0';
+    }
+};
+
+const configuration = OneSignal.createConfiguration({
+    authMethods: {
+        app_key: {
+        	tokenProvider: app_key_provider
+        }
+    }
+});
+const client3 = new OneSignal.DefaultApi(configuration);
+
+app.post('/sendToDevice3', async function(req, res){
+    var gmails = req.body.gmails; // gmail is a array
+    var notificationType = req.body.notificationType;
+    var numberOfType = parseInt(xx);
+    console.log(numberOfType);
+    switch (numberOfType) {
+        case 0:
+            notificationType = "you have an review";
+            break;
+        case 1:
+            notificationType = "your reported comment is approved";
+            break;
+        case 2:
+            notificationType = "your reported facility is addressed";
+            break;
+        case 3:
+            notificationType = "a new facility has been added to the app";
```

Comment:
Codacy has a fix for the issue: [Delete `······`](https://app.codacy.com/gh/CPEN321HELP/CPEN321/pullRequest?prid=9852520)
```suggestion
      notificationType = "a new facility has been added to the app";
```